test/event_test.py

```diff
@@ -559,23 +558,15 @@ def test_wait(self):
         self.assertEqual(wait_event.type, event.type)
 
         # Test case with timeout and no event in the queue.
-        if SDL1:
-            with self.assertRaises(TypeError):
-                pygame.event.wait(250)
-        else:
-            wait_event = pygame.event.wait(250)
-            self.assertEqual(wait_event.type, pygame.NOEVENT)
+        wait_event = pygame.event.wait(250)
+        self.assertEqual(wait_event.type, pygame.NOEVENT)
 
         # Test case with timeout and an event in the queue.
-        if SDL1:
-            with self.assertRaises(TypeError):
-                pygame.event.wait(250)
-        else:
-            event = pygame.event.Event(EVENT_TYPES[0], **EVENT_TEST_PARAMS[EVENT_TYPES[0]])
-            pygame.event.post(event)
-            wait_event = pygame.event.wait(250)
+        event = pygame.event.Event(EVENT_TYPES[0], **EVENT_TEST_PARAMS[EVENT_TYPES[0]])
```

Comment:
Nice the wait works with SDL1 :)

Reply:
That’s another advantage of doing the custom wait implementation :)